### ISSUE TITLE
Missing dependency, "puma" in web image - can't run "docker-compose up"

### ISSUE THREAD

**Issue by wolfch** [SUSE/Portus#429]
I did a "git pull" to get the October 12 (2015) changes where a container was added to run "cronos", but upon running "docker-compose up", I received the following error(s):

bash-4.1# docker-compose up
Removing portus_web_1...
portus_db_1 is up-to-date
Recreating be418e7932_be418e7932_portus_web_1...
Cannot start container 992df1d87896d422d4c7e647d9fcd773b18ca1b72a7f61ddec7f1d4dadc0cb3c: [8] System error: exec: "puma": executable file not found in $PATH

I am testing on RHEL-6.7 no Ruby or Rails installed.   If the idea is to supply Ruby & Rails, etc. in the container, then maybe "Puma" should also be added to the image build process.

**Comment by flavio** [comment]
Puma has been added this morning to the web image. Make sure you rebuild the images via `docker-compose build`.

Feel free to reopen the issue if that doesn't solve the problem.

**Comment by wolfch** [comment]
After running "docker-compose build" followed by "docker-compose up", I let it sit for over ten minutes to allow cronus to sync.  Then when I tried to access Portus UI, I got an error, "ActiveRecord::PendingMigrationError" with detail message, "Migrations are pending. To resolve this issue, run: bin/rake db:migrate RAILS_ENV=development".  I did nsenter to get a shell in the "portus_web" container but there was no rake executable.  I'm not really a rails/ruby guy, so not certain is that was an appropriate fix attempt.
